topics/ecology/tutorials/panoply_ebv/tutorial.md

```diff
@@ -20,8 +20,8 @@ tags:
 - interactive-tools
 contributors:
 - yvanlebras
-- ColineRoyaux
-- Marie59
+- colineroyaux
+- marie59
 - annefou
```

Comment:
If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.